NEMO/apps/kiosk/templates/kiosk/kiosk.html

```diff
@@ -322,6 +342,31 @@ <h3 id="interlock_message" class="text-center"></h3>
 		{
 			ajax_get('{% url 'kiosk_occupancy' %}', {'occupancy': occupancy}, function(response) { $("#occupancy").html(response); });
 		}
+        {# This function modifies the contents of a style sheet after the page has been loaded. #}
+        function set_style(css_text)
+        {
+            let sheet = document.createElement('style');
+            sheet.type = 'text/css';
+            (document.head || document.getElementsByTagName('head')[0]).appendChild(sheet);
+            return (set_style = function(css_text, node)
+            {
+                if(!node || node.parentNode !== sheet)
+                    return sheet.appendChild(document.createTextNode(css_text));
+                node.nodeValue = css_text;
+                return node;
+            })(css_text);
+        }
+
+        function fetch_usage()
+		{
+            $("#usage_title").html("<h2>Tools in use</h2>")
+			ajax_get('{% url 'kiosk_usage' %}', {'interest': 'tools'}, function(response) { $("#usage").html(response); });
+            set_style('.in_use { display: table-row; } .idle { display: none; }');
```

Comment:
I might be missing something here but wouldn't be easier to use jquery in the callback `function(response)` to show/hide stuff?
for example `$(".in_use").css("display", "table-row");` and `$(".idle").hide()`

Reply:
Good point, thanks, I changed it!